_ja/functions/filterNot.md

```diff
@@ -0,0 +1,11 @@
+---
+name: filterNot
+---
+
+# `filterNot`
+
+@include [signatures/filterNot.md]
+
+`filter` は、このコレクションの要素で述語 `p` を満たさないものを残し、残りを捨てたコレクションを作成します。
```

Comment:
Wrong function name.

Reply:
Fixed in https://github.com/superruzafa/visual-scala-reference/pull/11/commits/5d802388170b18a0bf3b64bcacb31fbcfb97c371